Fix auto dropdown for tag on page load

surajshetty3416 opened the pull request:
![tagfix2](https://user-images.githubusercontent.com/13928957/39701629-8791d43e-521f-11e8-879d-97b7f6c047f8.gif)


closes https://github.com/frappe/frappe/issues/5483

surajshetty3416 committed:
Fix auto dropdown for tag on page load

netchampfaris commented:
https://github.com/frappe/erpnext/wiki/Contribution-Guidelines

Screenshot? GIF?

surajshetty3416 committed:
Fix tag related bugs

- reset doc user_tags on change
- move "label conversion toTitle" from onTagAdd listener
to addTag function
- remove unnecessary refresh call
